templates/none/routes/index.js

```diff
@@ -0,0 +1,10 @@
+var express = require('express');
+var router = express.Router();
+
+router.get('/', function (req, res, next) {
+  res.sendFile('index.html', {
```

Comment:
Yea, there just wouldn't be a `/` route at all, since `express.static` is enabled to check for `index.html` for a directory request by default.